src/pineko/check.py

```diff
@@ -67,6 +67,31 @@ def check_grid_and_eko_compatible(pineappl_grid, operators, xif):
         raise ValueError("x grid in pineappl grid and eko operator are NOT compatible!")
 
 
+def is_fonll_b(fns, lumi):
+    """Check if the fktable we are computing is a DIS FONLL-B fktable.
+
+    Parameters
+    ----------
+        fns : str
+            flavor number scheme (from the theory card)
+        lumi : list(list(tuple))
+            luminosity info
+
+    Returns
+    -------
+            : bool
+            true if the fktable is a FONLL-B DIS fktable
+    """
+    for lists in lumi:
+        for el in lists:
+            if el[1] != 11:
```

Comment:
Oh yes sure. Just to understand, are there grids with other leptons at the moment?

Reply:
Yes, for CC it's very relevant (because they decide which W boson is, and so which flavors/antiflavors are probed).

They are specified through a member of the observable card:
```yaml
projectileDIS: ["electron", "positron", "neutrino", "antineutrino"]
```